test/blocks/global-navigation/keyboard/keyboard-mobile.test.js

```diff
@@ -0,0 +1,343 @@
+/* eslint-disable no-restricted-syntax */
+import { expect } from '@esm-bundle/chai/esm/chai.js';
+// eslint-disable-next-line import/extensions,import/no-relative-packages
```

Comment:
Same here. Relative imports directly from the node modules is a big anti-pattern, the linter correctly highlights this